build.sh

```diff
@@ -74,11 +103,14 @@ for PKG_PATH in $(catkin_topological_order --only-folders); do
 
   # dpkg-source-opts: no need for upstream.tar.gz
   sbuild --chroot-mode=unshare --no-clean-source --no-run-lintian \
-    --dpkg-source-opts="-Zgzip -z1 --format=1.0 -sn" --build-dir=/home/runner/apt_repo \
-    --extra-package=/home/runner/apt_repo "$@"
+    --dpkg-source-opts="-Zgzip -z1 --format=1.0 -sn" --build-dir=${APT_REPO} \
```

Comment:
revert this as well, please.

Reply:
Can you elaborate why it should be reverted? Or you mean attribution? I think it's correct, but I'll update the author to point to you.

Reply:
You mean the change replacing `/home/runner/apt_repo` with `${APT_REPO}`, right?

Reply:
yes.